charts/pre-api/values.stg.template.yaml

```diff
@@ -1,6 +1,6 @@
-# Don't modify this file, it is only needed for the pipeline to set the image and ingressHost
 java:
   environment:
     TESTING_SUPPORT_ENDPOINTS_ENABLED: true
+  # Don't modify this file beyond this line, it is only needed for the pipeline to set the image and ingressHost
```

Comment:
commented line should stay at the top

Reply:
I think the file is ok to edit though? it's just not to edit the bottom lines whereas the initial comment at the top suggests the whole file shouldn't be changed?